tests/backup/mysql/inventory_mysql_test.js

```diff
@@ -166,13 +167,13 @@ Scenario(
 );
 
 Scenario(
-  'PMM-T848 Verify service no longer exists error message during restore @bm-mysql @not-ui-pipeline',
+  '@PMM-T848 Verify service no longer exists error message during restore @bm-mysql @not-ui-pipeline',
```

Comment:
Is there any reason for adding @ to the test name?

Reply:
yes, it converts the ticket id in a tag and gives ability to run specific tests on pipeline and save some time on waiting for results